ruby/red-arrow/test/test-list-array.rb

```diff
@@ -25,6 +25,7 @@ class ListArrayTest < Test::Unit::TestCase
         [false, true, false],
       ]
       array = Arrow::ListArray.new(data_type, values)
+      assert_equal(data_type, array.value_data_type)
```

Comment:
I think that we don't need this assertion.
This is tests on C++ layer.

Reply:
Deleted.